myblog/blog/tests.py

```diff
@@ -124,4 +131,22 @@ def test_blogpost_api(self, client, user_1):
         response = client.get(url,format = 'json')
         assert len(response.data) == BlogPost.objects.count()
         
-        
\ No newline at end of file
+    def test_post_blogpost_api(self, user_1):
+        
+        url = reverse('api/post')
+        
+        # Login
+        client = APIClient()
+        client.login(username=user_1.username, password=user_1.password)
+        
+        client.force_authenticate(user=user_1)
+        
+        # Check db
+        assert BlogPost.objects.count() == 0 ; # Must be 0
```

Comment:
I don't know what were your intentions here, but my wild guess is that you forgot to check at the end of the test that there was a new blogpost created and thus the count increased.

Reply:
Yes, I forgot this!